accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,366 @@
+
+## Deploy a cluster using SUSE Manager
+
+Under SUSE Manager main menu, a new menu item "Cluster" should be introduced.
+Under that item, it should be possible to create, edit, and remove a cluster.
+
+To create a cluster, a user has to enter its name.
+
+Every detail about the cluster is saved into corresponding database tables (e.g. `susecluster`).
+
+### Cluster requirements
+
+If the cluster required a specific set of nodes optionally with a defined configuration, the cluster provider muster offer a way to deploy the whole infrastructure automatically, e.g. using AutoYaST profiles.
+
+SUSE Manager will provision bare-metal machines using the supplied AutoYaST profile to have a complete set of machines ready to be hosting the cluster.
+
+### The Management Node
+
+The cluster provider must also indicate:
+
+- how and the number of Management Node(s) must be elected among the provisioned machines (e.g. based on hardware requirements)
+- how Management Node(s) should be deployed (e.g. additional software to be installed via AutoYaST or Salt states)
+- whether Management Node(s) can be managed by SUSE Manager (specifically: can users manage the updates of the Management Node via SUSE Manager?)
+
+### Cluster bootstrapping
+
+The cluster must be initiated by the Management Node that has all the tools to bootstrap and manage the cluster. Specific instructions on how to bootstrap the cluster must be provided.
+Questions that need to be answered by the cluster provider:
+- Upon successful bootstrapping, can the worker nodes be managed by SUSE Manager?
+- Do the nodes have a specific agent running on them and/or can we install a `salt-minion` or use `salt-ssh` on those nodes?
+- Can the nodes be managed by SUSE Manager? Can the packages be updated or do they require specific handling by the Management Node?
+
+## Specific section: deploy a CaaS platform cluster using SUSE Manager
+
+  ### Cluster Deployment
+
+  Under SUSE Manager main menu, a new menu item "CaaS Platform" should be introduced.
+
+  Under that item, it should be possible to create, edit, and remove a cluster.
+
+  To create a cluster, a user has to enter its name.
+
+  Then, the user will be requested to associate a load balancer and a management node to the cluster.
+
+  The cluster is then identified by:
+  - a unique name
+  - the hostnames of the load balancers
+  - the hostname of its management node
+
+  (more on the load balancer and the management node below).
+
+  Every detail about the cluster is saved into corresponding database tables (e.g. `susecaaspcluster`).
+
+  NOTE: the load balancer is unique for the cluster and it is a level 4 load balancer (no TLS termination). The management node can manage different clusters at the same time.
```

Comment:
It was pointed out in https://github.com/uyuni-project/uyuni-rfc/pull/10#discussion_r311578112 
@ereslibre: what's your opinion?

Reply:
No need to be specific, but L4 should suffice and L7 will come with implications to the handler of the LB, since it will need to perform TLS termination and hence certificate renewal, revocation...

Reply:
Ok leaving as it is as per previous comment.